src/utils/TopicsHelper.js

```diff
@@ -1,85 +1,30 @@
-import footballIcon from '../res/images/targets/target-football.png';
-import travelIcon from '../res/images/targets/target-travel.png';
-import politicsIcon from '../res/images/targets/target-politics.png';
-import artIcon from '../res/images/targets/target-art.png';
-import datingIcon from '../res/images/targets/target-dating.png';
-import musicIcon from '../res/images/targets/target-music.png';
-import moviesIcon from '../res/images/targets/target-movies.png';
-import seriesIcon from '../res/images/targets/target-series.png';
-import foodIcon from '../res/images/targets/target-food.png';
-
-export const getTopicId = (name) => {
-  switch (name) {
-    case 'Football':
-      return 2;
-    case 'Travel':
-      return 3;
-    case 'Politics':
-      return 4;
-    case 'Art':
-      return 5;
-    case 'Dating':
-      return 6;
-    case 'Music':
-      return 7;
-    case 'Movies':
-      return 8;
-    case 'Series':
-      return 9;
-    case 'Food':
-      return 10;
-    default:
-      return 0;
+export const getTopicId = (name, topicsList) => {
+  let id;
+  for (let i = 0; i < topicsList.length; i++) {
```

Comment:
Use `forEach()`  instead of `for` in order to avoid out of range errors.
http://www.w3schools.com/jsref/jsref_forEach.asp